main_app/models.py

```diff
@@ -1,3 +1,24 @@
 from django.db import models
+from django.urls import reverse
 
-# Create your models here.
+class Business(models.Model):
+    name = models.CharField(max_length=100)
+    email = models.CharField(max_length=100)
+    location = models.CharField(max_length=100)
+    business_type = models.CharField(max_length=100)
```

Comment:
Can we change this to a choice field instead for business type?

Maybe have a select field of only [gym, yoga studio, wellness/dietician clinic] etc etc